languages.toml

```diff
@@ -2082,3 +2082,17 @@ indent = { tab-width = 2, unit = "  " }
 [[grammar]]
 name = "matlab"
 source = { git = "https://github.com/mstanciu552/tree-sitter-matlab", rev = "2d5d3d5193718a86477d4335aba5b34e79147326" }
+
+[[language]]
+name = "ponylang"
+scope = "source.pony"
+file-types = ["pony"]
+injection-regex = "pony"
+roots = ["corral.json", "lock.json"]
+auto-format = false
```

Comment:
If there is no language server or external formatter (`formatter` key) then the `auto-format` config has no effect